src/components/Books.js

```diff
@@ -1,18 +1,26 @@
 import BookList from './Booklist';
 import NewBook from './NewBook';
+import styles from '../styles/Books.module.css';
 
 function Books() {
   return (
     <>
       <div style={{
         display: 'flex',
-        flexDirection: 'row',
+        flexDirection: 'column',
         justifyContent: 'center',
         alignItems: 'center',
+        columnGap: '20px',
       }}
       >
-        <section style={{ display: 'flex', flexDirection: 'column' }}>
+        <section style={{
+          display: 'flex',
+          flexDirection: 'column',
+          padding: '30px',
+        }}
+        >
```

Comment:
- [`OPTIONAL`] Also, Please note that using inline comments is not recommended in React as it makes your codebase look clumsy and difficult to read. I would recommend you extra all your inline styles into an external CSS file to help improve the readability of your codebase.